src/main/java/net/snowflake/ingest/streaming/internal/StageMetadata.java

```diff
@@ -0,0 +1,117 @@
+package net.snowflake.ingest.streaming.internal;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+public class StageMetadata {
```

Comment:
mind sharing where this contract is coming from / how did you decide to come up with it ?

Reply:
It's from client configure ([ref](https://github.com/snowflakedb/snowflake/blob/46f48668a72e8f063bce1ac152616e16fd70d876/GlobalServices/src/main/java/com/snowflake/resources/streaming/StreamingResource.java#L1528))